wicket-spring/src/test/java/org/apache/wicket/spring/FieldBeansCollectorTest.java

```diff
@@ -40,7 +40,7 @@ public void resolveGenericList() throws Exception
 
 		FieldBeansCollector fieldBeansCollector = new FieldBeansCollector(resolvableType);
 
-		assertTrue(fieldBeansCollector.getFieldType() == FieldType.LIST);
+		assertEquals(fieldBeansCollector.getFieldType(), FieldType.LIST);
```

Comment:
arg positions also need to be be updated :)
are there any tools to check such errors? :)

`assertEquals(byte expected, byte actual, String message)`

Reply:
I'd been meaning to write or look for a recipe on that; "if either argument is a literal, that argument should come first".

Reply:
I'll let you know what I find & apply that here. :)

Reply:
Logged as a [possible improvement here](https://github.com/openrewrite/rewrite-testing-frameworks/issues/275). Once that's automated we can again apply it here, to also migrate any other imperfectly placed argument orders.

Reply:
Picked up in https://github.com/apache/wicket/pull/546